src/plugins/pendo/components/OptInModal/OptInModal.tsx

```diff
@@ -0,0 +1,159 @@
+          color='gray.500'
+          fontWeight='500'
+          fontSize='sm'
+        />
+        <Text translation='plugins.analytics.optInModal.actionsSentence' mb={4} fontWeight='500' />
+
+        <List borderRadius='xl' border={borderColor} borderWidth={'1px'}>
+          <ListItem
+            borderBottomWidth={'1px'}
+            p={4}
+            display='flex'
+            justifyContent='space-between'
+            alignItems='center'
+            flexDirection='row'
+          >
+            <Text
+              translation='plugins.analytics.optInModal.sendEvents'
+              fontSize='sm'
+              fontWeight='500'
+            />
+            <ListIcon as={IoMdCheckmark} mr={0} color='green.500' width='22px' height='22px' />
+          </ListItem>
+          <ListItem
+            borderBottomWidth={'1px'}
+            p={4}
+            display='flex'
+            justifyContent='space-between'
+            alignItems='center'
+            flexDirection='row'
+          >
+            <Text
+              translation='plugins.analytics.optInModal.neverCollectPersonalInformations'
+              fontSize='sm'
+              fontWeight='500'
+            />
+            <ListIcon as={IoMdClose} mr={0} color='red.500' width='22px' height='22px' />
+          </ListItem>
+          <ListItem
+            borderBottomWidth={'1px'}
+            p={4}
+            display='flex'
+            justifyContent='space-between'
+            alignItems='center'
+            flexDirection='row'
+          >
+            <Text
+              translation='plugins.analytics.optInModal.neverCollectIp'
+              fontSize='sm'
+              fontWeight='500'
+            />
+            <ListIcon as={IoMdClose} mr={0} color='red.500' width='22px' height='22px' />
+          </ListItem>
+          <ListItem
+            p={4}
+            display='flex'
+            justifyContent='space-between'
+            alignItems='center'
+            flexDirection='row'
+          >
+            <Text
+              translation='plugins.analytics.optInModal.neverSellData'
+              fontSize='sm'
+              fontWeight='500'
+            />
+            <ListIcon as={IoMdClose} mr={0} color='red.500' width='22px' height='22px' />
+          </ListItem>
+        </List>
+      </ModalBody>
+
+      <Divider mb={4} />
+
+      <ModalFooter flexDirection='column'>
+        <Button
+          isFullWidth
+          colorScheme={'blue'}
+          size='md'
+          onClick={handleConfirm}
+          data-test='consent-optin-continue-button'
+        >
+          <Text translation={'common.continue'} />
+        </Button>
+        <Button
+          isFullWidth
+          variant='ghost'
+          size='md'
+          mt={3}
+          onClick={() => (window.location.href = 'https://private.shapeshift.com')}
+          data-test='consent-optout-button'
+        >
+          <Text translation='plugins.analytics.optInModal.noThanks' />
```

Comment:
don't see the translation additions here, but is `Thanks` intended to be capitalized from the screenshot?